src/cli/argument-parser.ts

```diff
@@ -173,6 +173,7 @@ export default class CLIArgumentParser {
             .option('--no-color', 'disable colors in command line')
 
             // NOTE: temporary hide experimental options from --help command
+            .addOption(new Option('--proxyless', 'experimental').hideHelp())
```

Comment:
Let's rename as it's customary in Node.js: `--experimental-proxyless` and use the `experimentalProxyless` term in all places.
See also https://nodejs.org/api/cli.html#cli_experimental_abortcontroller.

Reply:
It's the public experimental Node API, but the `proxyless` is our internal flag. When we will be testing this flag is easier to write `--proxyless` than `--experimental-proxyless`. And when we will decide to make public this option, we will rename it. 😊

Reply:
Ok. Let it be.